### DISCUSSION TITLE
Use as telestrator : 2 missing features

### DISCUSSION THREAD

**Discussion by wasabi1809** [tableaunoir/tableaunoir#150] [Ideas]
Firstly thanks for the great app !

I have a special use case I guess (see below for details) and it’s so close to be a perfect solution...
Basically I miss (or couldn’t find):

1) a key bindings to clear the board without asking confirmation 
2) a possibility to hide the mouse cursor.

Now more info about what I want to do for the one interested :-) : 

I want to use it as a telestrator (ie: where I can write on videos, ...) to do this, I run the tableaunoir app (hiding the toolbar / full screen) on a raspberry pi connected to a wacom one (which has a screen), the wacom one hdmi input is connected to my ATEM program feed (video switcher) and so does the raspberry pi screen output, I add the raspberry pi screen as a downstream key (basically a layer on top of the video) and remove the black with a luma key... now if I write anywhere on the screen it goes on top of the video...  I use a streamdeck to send keyboard commands to the app (hence needing keyboards shortcuts)

Thanks again !

**Comment by francoisschwarzentruber** [comment]
Thank you for your message and your ideas! For the moment, these features are indeed not implemented. I suggest the following: I will implement the basics so that you may use it with a "script" (see menu). What do you think?

**Comment by wasabi1809** [comment]
It would be great but can this be triggered with a keyboard shortcut ? the raspberry pi is out of sight and has no keyboard connected, I use Vicreo to send key shortcuts through the network... and i don’t want to have to use the interface since this would occult the program feed itself (ie what i am writing on)

thanks and happy easter !

**Reply by francoisschwarzentruber** [reply]
Happy easter to you too!
1. You can now clear the board by pressing Ctrl + Alt + K
2. You can hide/show the mouse cursor by pressing Ctrl + Alt + H (also available in the option menu)
Hope it suits your need!

**Comment by wasabi1809** [comment]
Cool, I’ll try that asap... Thanks a lot !

**Comment by francoisschwarzentruber** [comment]
Dear wasabi1809, does it work properly?

**Comment by wasabi1809** [comment]
yes, its perfect, i use it daily... thanks

**Reply by francoisschwarzentruber** [reply]
Great! :)

**Comment by francoisschwarzentruber** [comment]
Hello,
for clearing the board, it now Alt + N (for new). It is the same functionnality as creating a new clear slide in a presentation. So  Ctrl + Alt + K is not working anymore. Hope it is still ok for you!

**Comment by wasabi1809** [comment]
yes, thanks for letting me know!
